src/test/e2e/api/client/feature.e2e.test.ts

```diff
@@ -60,11 +60,6 @@ beforeAll(async () => {
         'featureY',
         { feature: 'featureX', variants: ['featureXVariant'] },
     );
-    // depend on parent being disabled
```

Comment:
currently we only support only one dependency in admin. Will bring this back when we support more than one.